### PR TITLE
OKTA-462156 l10n error on password expiry warning

### PR DESCRIPTION
**Pull request by lesterchoi-okta**
### Bug:
Customer reports seeing `L10N_ERROR` instead of the correct warning message about an expiring password. This bug only occurs when the widget is on a custom URL domain (vs. Okta-hosted).

### Expected:
Widget displays the message "When your password expires you will be locked out of your {company_name} account."

### Actual:
Widget displays the message: "L10N_ERROR[idx.password.expiring.message]"

### Cause:
An unexpected i18n param in the IDX API response for introspect causes the widget to display  on customer-hosted widget.

Okta-hosted returns:
https://github.com/okta/okta-signin-widget/blob/b9d2e70e911168f1da95b7bb14ae9cc15121fc52/playground/mocks/data/idp/idx/authenticator-expiry-warning-password.json#L150-L164

Customer-hosted returns:
https://github.com/okta/okta-signin-widget/blob/b9d2e70e911168f1da95b7bb14ae9cc15121fc52/playground/mocks/data/idp/idx/authenticator-expiry-warning-password-with-brand-name.json#L150-L164

l10n:
https://github.com/okta/okta-signin-widget/blob/b9d2e70e911168f1da95b7bb14ae9cc15121fc52/packages/%40okta/i18n/src/properties/login.properties#L898

### Fix:
- ~~Option 1: update the API to use a new i18n key that expects a param for company name + send "Okta" as default param~~
- **Option 2: update the API to remove the company name param, and re-word message.**
- ~~Option 3: update the SIW (or i18n lib?) to handle the case conditionally~~

## PR Checklist

- [ ] Have you verified the basic functionality for this change?

- [ ] Added unit tests?

- [ ] Added e2e tests

- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?

- Did you verify the change by running downstream monolith artifacts? (YES | NO | UNSURE)

### Screenshot/Video:

localhost_3000_

### Issue:

- [OKTA-462156](https://oktainc.atlassian.net/browse/OKTA-462156)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "private": true,
   "name": "@okta/okta-signin-widget",
   "description": "The Okta Sign-In Widget",
-  "version": "6.2.0",
+  "version": "6.3.0",
```
Comment (aarongranick-okta) [inline package.json:5]: this should not be here. If you want it based on 6.2, you must rebase interactively and remove unrelated commits from master

File: playground/mocks/config/responseConfig.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ const idx = {
   ],
 
   '/idp/idx/introspect': [
-    'identify',
+    // 'identify',
```
Comment (aarongranick-okta) [inline playground/mocks/config/responseConfig.js:21]: remember to revert this change before you open PR for review

File: package.json (hunk 1 above)
Comment (lesterchoi-okta) [reply package.json:5]: Yeah i see the other unrelated. I'll rebase and update

File: playground/mocks/config/responseConfig.js (hunk 2 above)
Comment (lesterchoi-okta) [reply playground/mocks/config/responseConfig.js:21]: Will do. Enabled just the relevant responses for easy bug repro

**Comment by lesterchoi-okta** [comment]
Closing. Fix recommendation is to ask customer to pass `brandName` to prevent error caused by unexpected API param in IDX response.
https://oktainc.atlassian.net/browse/OKTA-462156?focusedCommentId=2487219
